javascript/javascript1/week2/fullName.js

```diff
@@ -0,0 +1,13 @@
+function getFullName(firstName, lastName, useFormalName) {
+    let name1 = firstName;
+    let name2 = lastName;
+
+    if ((useFormalName = true)) {
```

Comment:
Remember a single `=` is for assigning a variable. Use `===` for comparison: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators